servicedirectory/create_endpoint.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package servicedirectory
+
+// [START servicedirectory_create_endpoint]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	servicedirectory "cloud.google.com/go/servicedirectory/apiv1beta1"
+	sdpb "google.golang.org/genproto/googleapis/cloud/servicedirectory/v1beta1"
+)
+
+func createEndpoint(w io.Writer, projectId string) error {
```

Comment:
```suggestion
func createEndpoint(w io.Writer, projectID string) error {
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done for all argument and local variables